src/util/child.js

```diff
@@ -94,8 +94,20 @@ export function spawn(
           processingDone = true;
         }
 
-        proc.on('close', (code: number) => {
-          if (code >= 1) {
+        proc.on('close', (code: number, signal: string) => {
+          if (signal) {
+            err = new SpawnError(
+              [
+                'Command failed.',
```

Comment:
Looks like this message generation part is repeating itself quite a bit. May be you can DRY this up a bit?